ospfd/ospf_bfd.c

```diff
@@ -169,6 +169,10 @@ void ospf_interface_bfd_show(struct vty *vty, const struct interface *ifp,
 			     struct json_object *json)
 {
 	struct ospf_if_params *params = IF_DEF_PARAMS(ifp);
+
+	if (params == NULL)
+		return;
```

Comment:
In most other functions, we don't do this check! so it is either that this check isn't needed, or other functions need fixing too!